### PR TITLE
update osc configs to 4.11

### PR DESCRIPTION
**Pull request by cpmeadors**
Existing configuration was a POC and was never complete.  I have updated OCP versions and golang versions to represent the lastest release.  I have also removed the __4.9 variant config as it is not needed.

Closes: [KATA-1842](https://issues.redhat.com//browse/KATA-1842)

### CODE REVIEW

File: ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-master.yaml [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
-      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
   to: sandboxed-containers-operator
 operator:
   bundles:
   - as: sandboxed-containers-operator-bundle
     dockerfile_path: bundle-custom.Dockerfile
   substitutions:
-  - pullspec: controller:latest
+  - pullspec: proxy.engineering.redhat.com/rh-osbs/openshift-sandboxed-containers-operator
```
Comment (jensfr) [inline ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-master.yaml:27]: should that be registry-proxy.engineering.redhat.com instead?

Comment (cpmeadors) [reply ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-master.yaml]: yes

**Review by jensfr — APPROVED** [review]
lgtm, thanks @cpmeadors

**Pull request merged**